### PR TITLE
Adding some Python mode examples

### PR DESCRIPTION
**Pull request by villares**
I hope this is right...

### CODE REVIEW

**Comment by jeremydouglass** [comment]
I’m going to drop the credit lines for Java sketch authors (such as me) from the pyde files if that’s okay with you. The spirit of Rosetta code is that everybody copies everybody into different languages, but we’re including credit lines for each person that contributed actual Python code for the pyde example – in these examples, that is you!

**Comment by jeremydouglass** [comment]
PythagorasTree includes a standard sketch.properties file, but none of the others do. Should I copy that file into the other examples, or leave it out of all of them?

File: examples/Python/ColorWheel/ColorWheel.pyde [hunk 1]
```diff
@@ -0,0 +1,14 @@
+size(300, 300)
```
Comment (jeremydouglass) [inline examples/Python/ColorWheel/ColorWheel.pyde:1]: Just noting here that ColorWheel.pyde is missing a header. I've already copied it over from ColorWheel.pde (and fixed the title there).

File: examples/Python/Animation/Animation.pyde [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""
+Animation
+https://rosettacode.org/wiki/Animation#Processing
+Processing 3.4
+2019-08 Jeremy Douglass
+2020-03 Alexandre Villares (Python Mode)
+Task: Create a window containing the string "Hello World! "
```
Comment (jeremydouglass) [inline examples/Python/Animation/Animation.pyde:7]: Minor note -- in my merge I am now changing the spacing on all the "Task:" headers to match the linebreaks in Java examples and the wiki style. Compact is nice too, I just think it is easier if everything is the same way everywhere.

These aren't requests on your PR -- just notes on my changes.

**Comment by villares** [comment]
> PythagorasTree includes a standard sketch.properties file, but none of the others do. Should I copy that file into the other examples, or leave it out of all of them?

Sorry for that. I think we should leave them out. Maybe even `gitignore` them

File: examples/Python/MandelbrotSet/MandelbrotSet.pyde [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+https://rosettacode.org/wiki/Mandelbrot_set#Processing
+Processing 3.4
+2017-09-23 Noel
+2020-03-13 Alexandre (Python Mode)
+Task: Generate and draw the Mandelbrot set.
+Note that there are many algorithms to draw Mandelbrot set
+and there are many functions which generate it .
+"""
+
+# Click on an area to zoom in.
```
Comment (jeremydouglass) [inline examples/Python/MandelbrotSet/MandelbrotSet.pyde:12]: Thanks for doing this! Basically the style is that the wiki task instructions all go in the header comments, and any notes by the sketch author about the specific solution go in line comments.

**Review by jeremydouglass — COMMENTED** [review]
Everything is nice and clean!

File: examples/Python/ColorWheel/ColorWheel.pyde (hunk 1 above)
Comment (villares) [reply examples/Python/ColorWheel/ColorWheel.pyde:1]: thank you!

**Comment by jeremydouglass** [comment]
Merged as 46291cf

**Comment by jeremydouglass** [comment]
> Maybe even `gitignore` them

Good idea. I missed that in the release, but I'll add that to master now.

**Pull request closed without merging**